docs/usage/ipv6.md

```diff
@@ -0,0 +1,156 @@
+Therefore, they also expose their services only via load balancers with IPv4 addresses.
+To allow external clients to also use IPv6 to access services in an IPv4 shoot cluster, the cluster needs to be configured to support dual-stack ingress.
+
+It is possible to configure a shoot cluster to support dual-stack ingress, see [Using IPv4/IPv6 (dual-stack) Ingress in an IPv4 single-stack cluster](dual-stack-ingress.md) for more information.
+
+The main benefit of this approach is that the existing cluster stays almost as is without major changes, keeping the operational simplicity.
+It works very well for services that only require incoming communication, e.g. pure web services.
+
+The main drawback is that certain scenarios, especially related to IPv6 callbacks, are not possible.
+This means that services, which actively call to their clients via web hooks, will not be able to do so over IPv6.
+Hence, those services will not be able to allow full-usage via IPv6.
+
+## IPv6-only Shoot Clusters
+
+### Motivation
+
+IPv6-only shoot clusters are the best option to verify that services are fully IPv6-compatible.
+While [Dual-Stack Shoot Clusters](#dual-stack-shoot-clusters) may fall back on using IPv4 transparently, IPv6-only shoot clusters enforce the usage of IPv6 inside the cluster.
+Therefore, it is recommended to check with IPv6-only shoot clusters if a workload is fully IPv6-compatible.
+
+In addition to being a good testbed for IPv6 compatibility, IPv6-only shoot clusters may also be a desirable eventual target in the IPv6 migration as they allow to support both IPv4 and IPv6 clients while having a single-stack with the cluster.
+
+### Creating an IPv6-only Shoot Cluster
+
+To create an IPv6-only shoot cluster, the following needs to be specified in the `Shoot` resource (see also [here](usage.md#example-shoot-manifest-ipv6)):
+
+```yaml
+kind: Shoot
+apiVersion: core.gardener.cloud/v1beta1
+metadata:
+  ...
+spec:
+  ...
+  networking:
+    type: ...
+    ipFamilies:
+      - IPv6
+  ...
+  provider:
+    type: aws
+    infrastructureConfig:
+      apiVersion: aws.provider.extensions.gardener.cloud/v1alpha1
+      kind: InfrastructureConfig
+      networks:
+        vpc:
+          cidr: 192.168.0.0/16
+        zones:
+          - name: ...
+            public: 192.168.32.0/20
+            internal: 192.168.48.0/20
+```
+
+> [!WARNING]
+> Please note that `nodes`, `pods` and `services` should not be specified in `.spec.networking` resource.
+
+In contrast to that, it is still required to specify IPv4 ranges for the VPC and the public/internal subnets.
+This is mainly due to the fact that public/internal load balancers still require IPv4 addresses as there are no pure IPv6-only load balancers as of now.
+The ranges can be sized according to the expected amount of load balancers per zone/type.
+
+### Load Balancer Configuration
+
+The AWS Load Balancer Controller is automatically deployed when using an IPv6-only shoot cluster.
```

Comment:
The AWS Load Balancer Controller is automatically deployed when using an IPv6 or dual stack shoot cluster.